src/driver/collection.ts

```diff
@@ -100,6 +124,10 @@ export class Collection extends MongooseCollection {
   }
 
   updateOne(filter: Record<string, any>, update: Record<string, any>, options?: UpdateOneOptions) {
+    if (options != null) {
+      processSortOption(options);
+    }
+    console.log('UpdateOne', this.collection.updateOne.toString())
```

Comment:
I guess, `console.log` missed to be removed?

Reply:
Correct, sorry about that. Fixed in 9ed105b

Reply:
No problem